### PR TITLE
Update client.py

### PR DESCRIPTION
**Pull request by ntua-el20184**

### CODE REVIEW

**Comment by manonthegithub** [comment]
Hey @ntua-el20184 it seems that you are using the ChatGPT for random code generation. Please don't do it in our repos. 
I don't see any changes in 
`__get_extensions` and `create_dataset` functions. the `download` function was not modified, but just one more function was written so not there are two `download` functions.
Please check the code before submitting. Please also add tests for all the changes you submit. I will close this PR. Please feel free to open a new one containing tests and real changes to the functions.

**Comment by JJ-Author** [comment]
@manonthegithub this PR is related to #6, @ntua-el20184 forgot to mention it here but in the comment

**Comment by manonthegithub** [comment]
@JJ-Author I now that. Please read the comment, it is not because the PR is not linked to an Issue

**Comment by JJ-Author** [comment]
I agree that tests would be good, but I wanted to test it myself first. did you? also @ntua-el20184 should have the chance to reply to our question and revise his PR. just closing a PR without any chance to do so seems not a polite way, especially since it is my issue and I started reviewing the PR

**Comment by manonthegithub** [comment]
I did the review, it is not from nothing, my comment

File: databusclient/client.py [hunk 1]
```diff
@@ -25,6 +28,94 @@ class DeployLogLevel(Enum):
...
+ARTIFACT_IDENTIFIER = "artifact"
+VERSION_IDENTIFIER = "version"
+FILE_IDENTIFIER = "file"
+USER_IDENTIFIER = "user"
+
+def __download_files(urls: List[str], local_dir: str):
+    for url in urls:
+        __download_file__(url=url, filename=os.path.join(local_dir, wsha256(url)))
+
+def download(
```
Comment (manonthegithub) [inline databusclient/client.py:42]: the function overrides the download below in the code, it is not editing of the old function function, but writing a new one in the same file with the old one on line 461 in old version

**Comment by JJ-Author** [comment]
@ntua-el20184 prepped a commit for you to open another PR 
https://github.com/dbpedia/databus-python-client/commit/d3ca19cded0c8920f707497daee9f237ca751637
please incorporate changes into the download function where I added TODO comments
and address the issues mentioned from @manonthegithub 
especially adding tests to download each one of user, group, artifact, version, file

**Pull request closed without merging**